ci-operator/templates/openshift/openshift-azure/build-base-image.yaml

```diff
@@ -0,0 +1,224 @@
+      resources:
+        requests:
+          cpu: 1
+          memory: 300Mi
+        limits:
+          cpu: 3
+          memory: 4Gi
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp/shared
+      - name: cluster-secrets-azure
+        mountPath: /usr/share/ansible/openshift-ansible/inventory/dynamic/injected
+      env:
+      - name: HOME
+        value: /usr/share/ansible/openshift-ansible
+      - name: AZURE_VM_SIZE
+        value: "${AZURE_VM_SIZE}"
+      - name: JOB_NAME_SAFE
+        value: "${JOB_NAME_SAFE}"
+      - name: JOB_NAME_HASH
+        value: "${JOB_NAME_HASH}"
+      - name: DEPLOY_OS
+        value: "${DEPLOY_OS}"
+      - name: INSTANCE_PREFIX
+        value: ${NAMESPACE}-${JOB_NAME_HASH}
+      command:
+      - /usr/local/bin/entrypoint-provider
+      args:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        # error handling and sync code
+        set -euo pipefail
+
+        # We need to persist this variable so that we can clean it up when we exit
+        echo "RESOURCE_GROUP=${INSTANCE_PREFIX}-$(TZ=Etc/UTC date +%Y%m%d%H%M)" >>/tmp/shared/environment
+        echo "IMAGE_NAME=${DEPLOY_OS}-base-$(TZ=Etc/UTC date +%Y%m%d%H%M)" >>/tmp/shared/environment
+
+        trap 'rc=$?; if [[ $rc -ne 0 ]]; then
+          touch /tmp/shared/exit;
+        fi;
+        az group delete -n ${RESOURCE_GROUP} -y --no-wait
+        exit $rc' EXIT
+        trap 'kill $(jobs -p); exit 0' TERM
+
+        # Get resource group and image name
+        source /tmp/shared/environment
+        echo "RESOURCE_GROUP=${RESOURCE_GROUP}"
+        echo "IMAGE_NAME=${IMAGE_NAME}"
+
+        source /usr/share/ansible/openshift-ansible/inventory/dynamic/injected/secret
+        az login --service-principal -u ${AZURE_CLIENT_ID} -p ${AZURE_CLIENT_SECRET} --tenant ${AZURE_TENANT_ID} &>/dev/null
+        # Unfortunately az login is not enough for the node build playbook, we also
+        # have to source the following variables: AZURE_TENANT and AZURE_SECRET
+        # instead of AZURE_TENANT_ID and AZURE_CLIENT_SECRET which are used elsewhere.
+        cp /usr/share/ansible/openshift-ansible/inventory/dynamic/injected/secret /tmp/shared/credentials
+        sed -i -e "s#TENANT_ID#TENANT#" /tmp/shared/credentials
+        sed -i -e "s#CLIENT_SECRET#SECRET#" /tmp/shared/credentials
+        source /tmp/shared/credentials
+
+        ansible-playbook \
+          -e openshift_azure_resource_group_name=$RESOURCE_GROUP \
+          -e openshift_azure_resource_location=eastus \
+          -e openshift_azure_input_image_ns=images \
+          -e openshift_azure_input_image_prefix=${DEPLOY_OS}-root \
+          -e openshift_azure_output_image_ns=images \
+          -e openshift_azure_output_image_name=$IMAGE_NAME \
+          -e openshift_azure_vm_size=$AZURE_VM_SIZE \
+          playbooks/azure/openshift-cluster/build_base_image.yml
+
+        touch /tmp/shared/image_create
+
+    - name: tagimage
```

Comment:
tag after you create the image, no need for this container.

Reply:
alright